docs/locale/de/LC_MESSAGES/index.po

```diff
@@ -8,14 +8,18 @@ msgid ""
 msgstr ""
```

Comment:
These changes don't seem to belong here.

Reply:
I made a separate commit, i could also remove this commit from this PR, but let's see what @blocktrron thinks